api/db/migrations/20241206100509_create-legal-document-versions-user-acceptances.js

```diff
@@ -0,0 +1,27 @@
+const TABLE_NAME = 'legal-document-versions-user-acceptances';
+
+const up = async function (knex) {
+  await knex.schema.createTable(TABLE_NAME, function (table) {
+    table
+      .integer('documentId')
```

Comment:
Pour être parfaitement clair : 
```suggestion
      .integer('legalDocumentVersionId')
```